Src/AutoFixture/Fixture.cs

```diff
@@ -140,6 +140,9 @@ public Fixture(ISpecimenBuilder engine, MultipleRelay multiple)
                                 new TypeRelay(
                                     typeof(IDictionary<,>),
                                     typeof(Dictionary<,>)),
+                                new TypeRelay(
+                                    typeof(IReadOnlyDictionary<,>),
+                                    typeof(Dictionary<,>)),
```

Comment:
Is `Dictionary<,>` *read-only*? Why not [`ReadOnlyDictionary<,>`](https://msdn.microsoft.com/en-us/library/gg712875.aspx)? 😕